### PR TITLE
blog post: github integration

### PR DESCRIPTION
**Pull request by srinivasupadhya**

### CODE REVIEW

File: _posts/2015-06-30-GitHub-Integration.markdown [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+* Configure plugin
+
+    <figure>
+      <img src="/images/blog/authentication-plugins/configure-plugin.png" class="has_border full_size"
+        alt="Figure 2: GoCD - Configure Plugin" id="mature_ci_cd_setup" title="GoCD - Configure Plugin" />
+      <figcaption>Figure 1: Configure plugin pop-up <span class="click_to_enlarge">(Click to enlarge)</span></figcaption>
+    </figure>
+
+* Ability to Search & Add users
```
Comment (ketan) [inline _posts/2015-06-30-GitHub-Integration.markdown]: Would you mind stepping back a bit and describing where to get the OAuth keys?

Also can we call them "oauth client ID" and "oauth client secret". Would you prefer that I log a bug against https://github.com/srinivasupadhya/gocd-oauth-login?

**Comment by ketan** [comment]
Also, this blog feels more like "here are the things that can be done" as against "here is how to use these integrations" What is the intention of this blog?

**Comment by ketan** [comment]
Would it also make sense to mention a few of the things that can be done (along with a 2-4 screenshots), with pointers to the project home pages?

The blog feels very dry with not much of meat in it. Not that the integration is not a huge feat by itself :)

File: _posts/2015-06-30-GitHub-Integration.markdown (hunk 1 above)
Comment (srinivasupadhya) [inline _posts/2015-06-30-GitHub-Integration.markdown]: i will add the OAuth keys screenshot. its available in [here](http://www.go.cd/2015/06/18/authentication-end-point.html).

> Also can we call them "oauth client ID" and "oauth client secret"
yes, please log a bug so i remember to change it.

**Comment by srinivasupadhya** [comment]
> "here are the things that can be done" as against "here is how to use these integrations". What is the intention of this blog?

The intention is to show that users can tightly integrate with GitHub. How to do it is shown in quiet a bit of detail on project's README. I think those should be enough.

> The blog feels very dry with not much of meat in it.

I agree. But its a "info" blog vs a "feature" blog. Please feel free to make any improvements to it.

**Comment by ketan** [comment]
Blogs IMO should contain short pieces of information that should excite users. To know more about specifics, users can follow additional links to get to whatever it is that interests them.

> I agree. But its a "info" blog vs a "feature" blog. Please feel free to make any improvements to it.

I'm happy to make improvements, would you be OK with me chopping off content to make things a bit more exciting?

**Comment by srinivasupadhya** [comment]
> would you be OK with me chopping off content

sure. just keep the 3 points (authentication, PR builds & issue tracker) there. thanks.

**Comment by srinivasupadhya** [comment]
@ketan - added you as collaborator to my fork.

**Pull request merged**